components/runtimes/charliecloud/SPECS/charliecloud.spec

```diff
@@ -13,56 +13,61 @@
 # Base package name
 %define pname charliecloud
 
+# libexec path
+%define _libexecdir %{_prefix}/libexec
+
```

Comment:
I am bit unsure about this. Most files are installed in /opt/ohpc. This moves some files under /usr.

We have a few packages which are installed under /usr, but this is probably something new. Some files under /opt/ohpc some under /usr. If some other charliecloud package (without the -ohpc suffix) would be installed this files would probably collide.

Reply:
Good point. I've addressed this in my local branch by defining libexecdir as `%{install_path}/libexec`.